starterkits/primary/tailwind.config.js

```diff
@@ -1,18 +1,26 @@
 module.exports = {
   theme: {
     colors: {
-      'white': 'var(--color-1)',
-      'black': 'var(--color-2)',
-      'robins-egg-blue': 'var(--color-3)',
-      'turquoise': 'var(--color-4)',
-      'viking': 'var(--color-5)',
-      'blizzard-blue': 'var(--color-6)',
-      'onahau': 'var(--color-7)',
+      transparent: 'transparent',
+      white: 'var(--color-1)',
+      black: 'var(--color-2)',
+      charcoal: 'var(--color-3)',
+      'robins-egg-blue': 'var(--color-4)',
+      turquoise: 'var(--color-5)',
+      viking: 'var(--color-6)',
+      'blizzard-blue': 'var(--color-7)',
+      mabel: 'var(--color-8)',
+      red: 'var(--color-9)',
+      cosmos: 'var(--color-10)',
```

Comment:
maybe manage it with functional colors? this is error one.

Reply:
fixed